pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -840,6 +840,20 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 	return result
 }
 
+func cpuStatsInitialized(cpu *statsapi.CPUStats) bool {
+	return cpu != nil &&
+		(*cpu.UsageCoreNanoSeconds != 0 || *cpu.UsageNanoCores != 0)
```

Comment:
Wondering why we use the actual number instead of the `Time metav1.Time` of CPUStats for check whether the cpu is initialized ? 

Is there any cases that the values could be both zero but it did initialized (getting from the cadvisor of the container cpu stats that both values are zero) ?

Reply:
That's because we are trying to catch the corner case where cAdvisor doesn't actually know the usage stats (because it can't query cgroupfs for them) but still sends data as if it does. So the timestamp will always be set and we can't use it. 

In the case cAdvisor legitimately reports that the values are zero, we expect CRI stats to report the same, so in this case the end result is equivalent.